fails:Exception#to_s
fails:Exception#to_s returns the self's name if no message is set
